symphony-ws/startWithPostgres.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```

Comment:
This script was found here https://github.com/tonda100/wildfly-postgresql.

It start the wildfly server (without any war) and then connects to the wildfly cli in order to install/configure the postgres driver. It then stops the server.

It should then copy the war to the correct location (this somehow does not work currently) and then start the wildfly instance with the war.